src/frontend/Komponenter/Behandling/Vurdering/utils.ts

```diff
@@ -175,3 +175,9 @@ export const kopierBegrunnelse = (
         return nyeVurderinger;
     }
 };
+
+export const sjekkErInngangsvilkårType = (
```

Comment:
Kan denne renames til `kanVilkårGjenbrukes`?

Reply:
Funksjonen er for å sjekke om vilkåret er et inngangsvilkår, synes derfor at navnet beskrivende.

Reply:
Funksjonen brukes i to filer.

`const erInngangsvilkårType = sjekkErInngangsvilkårType(vurdering.vilkårType);`

```
                    {erInngangsvilkårType && (
                        <Button
                            onClick={gjenbrukVilkårsvurdering}
                            variant={'tertiary'}
                            type={'button'}
                        >
                            Gjenbruk
                        </Button>
                    )}
```

Tenkte det var best at variabelnavnet var kortest. Hva tenker du du?

Kan f.eks. endre til `const erInngangsvilkår = sjekkErInngangsvilkår(vurdering.vilkårType);` eller synes du det er bedre å bruke funksjonen direkte?

```
                    {erInngangsvilkår(vurdering.vilkårType) && (
                        ...
                    )}
```

Reply:
Endret

```
                            {skalViseGjenbrukKnapp(vurdering) && (
                                ...
                            )}
```